.yamato/promotion.yml

```diff
@@ -38,26 +38,6 @@ promotion_test_{{ platform.name }}_{{ editor.version }}:
 {% endfor %}
 {% endfor %}
 
-promotion_test_trigger:
```

Comment:
is this the test that was run after running the unit tests on all platforms but before promoting? the promotion test on 2019.1 I think

Reply:
The promotion test is that test.

The promotion test trigger is causing that test to run if you click on the promotion test trigger and queue it up. Which is pretty useless since I could have clicked on the promotion test and queued it up.